resources/db/schema/mysql-ddl.sql

```diff
@@ -1087,7 +1087,7 @@ CREATE TABLE user_movies (
   id          INT(16) UNSIGNED               NOT NULL AUTO_INCREMENT,
   user_id INT(16)                        NOT NULL,
   imdbid      MEDIUMINT(7) UNSIGNED ZEROFILL NULL,
-  categoryid  VARCHAR(64)                    NULL DEFAULT NULL,
+  categories_id  VARCHAR(64)                    NULL DEFAULT NULL,
```

Comment:
We don't currently have a standard for that, but I'd prefer plain 'categories' as the _ids would make it too similar to categories_id and more prone to mistakes. Also, rows are usually singular, because they only contain a single value, so having it plural without the _id tells you it contains more than one value.
All of this is irrelevant if Darius is correct and the field contains a single value.